test/regress/regress0/strings/str-rev-simple.smt2

```diff
@@ -0,0 +1,11 @@
+(set-logic ALL)
+(set-info :status sat)
+(declare-fun x () String)
+(declare-fun y () String)
+(declare-fun z () String)
+(assert (= (str.rev "ABC") "CBA"))
+(assert (= (str.rev "") ""))
+(assert (= (str.rev "A") x))
+(assert (= (str.rev (str.++ x y)) "BCA"))
+(assert (= (str.rev "BC") y))
+(check-sat)
```

Comment:
Maybe I am missing something but this looks like essentially the same benchmark as test/regress/regress1/strings/str-rev-simple-s.smt2 ? Is there a reason for having both? Also, shouldn't we require `--strings-exp` for `str.rev`?

Reply:
The last assertion is different: here, we say y is the reverse of "BC", in the other we say that y is some string whose reverse is "BC".  The latter is more difficult at the moment (without using an injectivity argument) and requires --string-exp.

Reply:
I see. Thanks for the explanation.